src/main/resources/view/MainWindow.fxml

```diff
@@ -12,7 +12,7 @@
 <?import javafx.scene.layout.VBox?>
 
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
-         title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+         title="Entrybook App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
```

Comment:
Small detail but since it's user facing, should we rename this to "Readme App" or just "Readme" instead? Or we can do it in another PR once we have settled on a name.